doc/configuration.md

```diff
@@ -7,14 +7,35 @@ can apply minor adjustments and still utilize the automatic build process. The f
 
 Setting name                 | Description                                                    | Default | Example
 -----------------------------|----------------------------------------------------------------|---------|----------------
-PRE\_BUILD\_COMMAND          | Command or a repo-relative path to a shell script to be run before build   | ""      | `echo foo`, `scripts/prebuild.sh`
-POST\_BUILD\_COMMAND         | Command or a repo-relative path to a shell script to be run after build    | ""      | `echo foo`, `scripts/postbuild.sh`
+PRE\_BUILD\_COMMAND          | Command to a shell script to be run before build   | ""      | `echo foo`
+PRE\_BUILD\_SCRIPT\_PATH     | A repo-relative path to a shell script to be run before build   | ""      | `scripts/prebuild.sh`
+POST\_BUILD\_COMMAND         | Command to a shell script to be run after build    | ""      | `echo foo`
+POST\_BUILD\_SCRIPT\_PATH    | A repo-relative path to a shell script to be run after build    | ""      | `scripts/postbuild.sh`
+ENABLE\_DYNAMIC\_INSTALL     | Enable dynamically install platform binaries if not presented inside the image | various | `true`, `false`
```

Comment:
what does various mean here?

Reply:
I put various here because it depends on which kind of image it is, like if it's a full build image it doesn't set to be true, if it's lts-versions or github-actions image then it was set to be true.